selenium_reading_docs.py

```diff
@@ -0,0 +1,51 @@
+from selenium import webdriver 
+from selenium.webdriver.common.by import By
+import openai
+import time
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+# apikey
+openai.api_key = "OPENAI_API_KEY"
+
+# starta o webdriver
+driver = webdriver.Firefox()
+
+# url pra acessar
+url = driver.get('https://g1.globo.com/')
+print("Acessando: ", driver.title)
+time.sleep(3)
+
+# wait pro elemento ficar clicavel
+wait = WebDriverWait(driver, 10)
+news_title = wait.until(EC.element_to_be_clickable((By.CSS_SELECTOR, ".feed-post-link")))
+
+# teste - scroll pra garantir que esteja visivel
+driver.execute_script("arguments[0].scrollIntoView(true);", news_title)
+
+# action de clique
+news_title.click()
+print("Notícia selecionada: ", news_title.text)
```

Comment:
selenium.common.exceptions.StaleElementReferenceException: Message: The element with the reference [...] is stale; either its node document is not the active document, or it is not the active document, or it is no longer connected to the DOM